app/controllers/projects_controller.rb

```diff
@@ -97,7 +95,7 @@ def project_params
   end
 
   def project
-    @project
+    current_project
   end
```

Comment:
you can use `alias_method` to get rid of this and keep the compatibility with the rest if you don't want to touch the rest of the code.